### DISCUSSION TITLE
Under review for a month

### DISCUSSION THREAD

**Discussion by imysun** [WalletConnect/walletconnect-monorepo#4474] [Developer Support]
Hi, team. I am the co-founder of Nest Wallet, a cross-chain wallet for traders. We submitted the app for explorer, but it has been under review for more than 3 weeks.

Here is our project ID: ab8ba260b8cc9a9b24746532b7a18799

Any help in getting our project reviewed and accepted would be greatly appreciated!!!

**Comment by glitch-txs** [comment]
reviewed! 👍
